radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/transport/PeerChannel.java

```diff
@@ -333,11 +334,11 @@ public boolean isOutbound() {
 	}
 
 	public String getHost() {
-		return this.host;
+		return remoteAddress.map(InetSocketAddress::getHostString).orElse("?");
 	}
 
 	public int getPort() {
-		return this.port;
+		return remoteAddress.map(InetSocketAddress::getPort).orElse(0);
```

Comment:
What's the rationale for these default values?

Reply:
just not to have to deal with a null/optional everywhere else